app/models/response.rb

```diff
@@ -128,6 +149,12 @@ def determine_student_mentor
     [student, mentor]
   end
 
+  def invitation_url
+    raise 'Cannot generate invitation_url for historical invitation tokens!' if invitation_token.token_plain.blank?
+    concatenated_token = "#{protocol_subscription.person.external_identifier}#{invitation_token.token_plain}"
+    "#{ENV['HOST_URL']}?q=#{concatenated_token}"
```

Comment:
`"#{ENV['HOST_URL']}?u=#{concatenated_token}"`?

Reply:
Die slash is dus niet nodig? Cool. Dat wist ik niet. Bespaart weer een karakter.

Reply:
idd